### PR TITLE
Add facenet tests from tensorflow_frozen to tensorflow, keras and mxnet

### PR DESCRIPTION
**Pull request by rainLiuplus**

### CODE REVIEW

File: mmdnn/conversion/coreml/coreml_emitter.py [hunk 1]
```diff
@@ -160,6 +160,12 @@ def _get_padding(IR_node):
 
         return pads
 
+    def emit_Mul(self, IR_node):
+        """
+        Skiped
+        """
+        pass
```
Comment (namizzz) [inline mmdnn/conversion/coreml/coreml_emitter.py]: why skip?

File: mmdnn/conversion/examples/mxnet/imagenet_test.py [hunk 2]
```diff
@@ -39,11 +39,11 @@ def print_result(self):
...
         self.preprocess(image_path)
 
-        # self.print_intermediate_result('pooling0', False)
+        self.print_intermediate_result('InceptionResnetV1/Repeat/block35_1/Conv2d_1x1/Conv2D', False)
 
-        self.print_result()
+        # self.print_result()
 
-        self.test_truth()
+        # self.test_truth()
```
Comment (namizzz) [inline mmdnn/conversion/examples/mxnet/imagenet_test.py]: Change it back

File: mmdnn/conversion/coreml/coreml_emitter.py (hunk 1 above)
Comment (rainLiuplus) [reply mmdnn/conversion/coreml/coreml_emitter.py]: It has not been implemented yet.

File: tests/test_conversion_imagenet.py [hunk 3]
```diff
@@ -328,7 +327,7 @@ def TensorflowEmit(original_framework, architecture_name, architecture_path, wei
         # import converted model
         model_converted = imp.load_source('TFModel', converted_file + '.py').KitModel(weight_path)
 
-        input_tf, model_tf = model_converted
+        input_tf,model_tf = model_converted
```
Comment (kitstar) [inline tests/test_conversion_imagenet.py]: add the space

**Pull request merged**